abci/proposals/proposals.go

```diff
@@ -198,7 +199,7 @@ func (h *ProposalHandler) PrepareProposalHandler() sdk.PrepareProposalHandler {
 // injectTx will only appear once regardless of how many times you attempt to inject it.
 // If injectTx is large enough, all originalTxs may end up being excluded from the returned tx array.
 func (h *ProposalHandler) injectAndResize(appTxs [][]byte, injectTx []byte, maxSizeBytes int64) [][]byte {
-	var returnedTxs [][]byte
+	returnedTxs := make([][]byte, 0, len(appTxs)) // init with initial capacity
```

Comment:
This is going to keep appending to the empty list instead. I dont think this needs to be preallocated?